cmplxfoil/CMPLXFOIL.py

```diff
@@ -411,7 +409,7 @@ def solveCL(
                 print("Converged!")
                 break
             elif abs(res > maxRes):
-                warnings.warn(f"solveCL diverged (CL = {cl:.2f})")
+                warnings.warn(f"solveCL diverged (CL = {cl:.2f})", stacklevel=2)
```

Comment:
What does this do?

Reply:
It dictates the number of levels of the stack trace to print. The new flake8-bugbear version makes you specify it (see B028 [here](https://github.com/PyCQA/flake8-bugbear)) and suggests at least 2 to make debugging easier.